browser.lisp

```diff
@@ -152,7 +152,39 @@
   "Move backwards a page."
   (webkit2:webkit-web-view-go-back (webview browser)))
 
+(defparameter *fallback-timeout* .2)
+
+;; 20150212 - MDC - Fixes issue with unavailable default scheme when opening url
```

Comment:
Comments like this are unnecessary, we have VCS for attributing reasoning to code.